public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -230,7 +223,7 @@ define(function(require) {
       // Listen for ESC to close
       _escKeyHandler = new KeyHandler.ESC(hidePublishDialog);
 
-      publisher.fsync.saveAndSyncAll(function(err) {
+      FileSystemSync.saveAndSyncAll(function(err) {
```

Comment:
We should add a comment above this explaining that this is only done as a precautionary measure to make sure that every time we make the publish dialog appear, we save unsaved files to the fs.